cmake/module/FindBerkeleyDB.cmake

```diff
@@ -27,13 +27,15 @@ This module defines the following variables:
 
 #]=======================================================================]
 
-if(BREW_COMMAND)
+set(_BerkeleyDB_homebrew_prefix)
+if(CMAKE_HOST_APPLE)
+  find_program(HOMEBREW_EXECUTABLE brew REQUIRED)
```

Comment:
Doesn't this make homebrew required even for depends builds?

Reply:
Thanks! Fixed.